kuma/static/styles/minimalist/structure/_article.scss

```diff
@@ -1,6 +1,9 @@
 /*
 Main wiki article content
 ====================================================================== */
+
+$clock-url: '../img/clock.svg';
```

Comment:
Changing this to the following should work here:

```suggestion
$clock-url: '../../../general/clock.svg';
```

Reply:
@schalkneethling it seems like it's not finding the icon located there. I tried to just do:

`$clock-url: '../../../general/clock.svg';`
and 
`$path-to-svg-general-icons + 'clock.svg';` because that variable was defined. The reason i moved the clock to the img folder was to follow [this](https://github.com/mdn/kuma/blob/6b6cb9f7648aba08ee797a1190973929cf13401a/kuma/static/styles/minimalist/structure/_footer.scss#L1) which works well. 

Let me know if you have any other input.

Reply:
So, this will work: `../general/clock.svg`

I noticed that the current SVG has the `width` and `height` attributes on it as well as `viewBox`. The problem was not introduced here, but I would ask that you remove the `width` and `height` attributes as well. 

Our icon set should also have a base color of `#333` but that is just a note to self. I also notice that this icon does *not* do well at `16px` so I should redraw this one as well.

Perhaps to make it render a bit better, for now, let’s set the font size of `.metadata ul` to `@include set-font-size(16px);` and then set the size of the icon to `18px`

Reply:
@schalkneethling 

> So, this will work: ../general/clock.svg

Correct, thank you 👍 

I didn't adjust the size of the icon because it looked good to me, when i made it `18px` it looked a little too big in my opinion.

16px:

![Screen Shot 2019-10-24 at 9 39 13 AM](https://user-images.githubusercontent.com/1201062/67491520-11008800-f643-11e9-87ce-317ed53465eb.png)
 
18px:

![Screen Shot 2019-10-24 at 9 45 21 AM](https://user-images.githubusercontent.com/1201062/67491527-15c53c00-f643-11e9-92c1-d565889a497a.png)

However, feel free to look over the changes and if you still want that size change just let me know!